tests/src/utility/installer_request_builder.rs

```diff
@@ -44,6 +45,13 @@ pub(crate) static TEST_CUSTOM_METADATA: Lazy<BTreeMap<String, String>> = Lazy::n
     attributes.insert("mythology".to_string(), "Nordic".to_string());
     attributes
 });
+pub(crate) static TEST_CUSTOM_UPDATED_METADATA: Lazy<BTreeMap<String, String>> = Lazy::new(|| {
+    let mut attributes = BTreeMap::new();
+    attributes.insert("deity_name".to_string(), "Baldur".to_string());
+    attributes.insert("mythology".to_string(), "Nordic".to_string());
```

Comment:
Irrelevant and pedantic, but pretty sure it is "Norse"...Finland is _Nordic_ but had a different mythology.